tests/Unit/Math/BigInteger/TestCase.php

```diff
@@ -358,4 +358,15 @@ public function testDiffieHellmanKeyAgreement()
             'Failed asserting that Alice and Bob share the same BigInteger.'
         );
     }
+
+    /*
```

Comment:
Can you try a double star here, i.e. `/**` instead of `/*`? That's how the example in https://phpunit.de/manual/current/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.requires.tables.api does it.